packages/-ember-data/tests/unit/model/relationships/has-many-test.js

```diff
@@ -2299,10 +2300,11 @@ module('unit/model/relationships - DS.hasMany', function(hooks) {
 
   test('DS.ManyArray is lazy', function(assert) {
     let peopleDidChange = 0;
+    let expectedNumberOfChanges = CUSTOM_MODEL_CLASS ? 1 : 2;
     const Tag = DS.Model.extend({
       name: DS.attr('string'),
       people: DS.hasMany('person'),
-      peopleDidChange: observer('people', function() {
```

Comment:
we should be very careful with this one, possibly confirm with some apps powering their selects with hasMany's that this doesn't break them.